### ISSUE TITLE
Colourblind-friendly plots

### ISSUE THREAD

**Issue by Tandethsquire** [Tandethsquire/emulatorr#6]
Most plots use one of the viridis colour palettes, which are chromatically safe for all forms of colourblindness. However, the 'standard' palette for implausibility (green shading to red) is definitely not. Need to come up with a new palette for implausibility that satisfies:
- Colourblind friendly: potentially the best extreme colours are purple and yellow
- Divergent in hsv so that greyscaled plots are still distinguishable
- Still gives the requisite information: should the lighter colour be for better points? How do we ensure that the implausibility plots are still obviously interpretable?

This should be offered as default, with potentially the option to switch to G/R palette (not the other way around!)

**Comment by Tandethsquire** [comment]
Provisional colourblind friendly plotting option added to ``emulator_plot(..., cb = TRUE)``: colour scale is three-colour diverging 1aff1a -> ac9011 -> 48118e. Will re-open and update if alternate colour schemes are found to be better. Before and after below.

![OriginalColour](https://user-images.githubusercontent.com/14112279/87322685-a02d3300-c525-11ea-8f7d-9036932c10fb.png)
![CBColour](https://user-images.githubusercontent.com/14112279/87322689-a0c5c980-c525-11ea-8b12-622e7c0766bd.png)

If a better colour scheme is found, will re-open and update accordingly.
